src/modules/forms.css

```diff
@@ -9,24 +9,15 @@ fieldset {
 }
 
 legend {
-  font-weight: 600;
```

Comment:
don't remove just change it back to a default, if we don't put anything the browser will decide